earthpy/tests/test_plot_rgb.py

```diff
@@ -154,3 +155,5 @@ def test_stretch_output_scaled(rgb_image):
         axs
     finally:
         del axs
+
+    plt.close()
```

Comment:
It looks like calling `plt.close()` with no arguments will only close the current figure (https://matplotlib.org/api/_as_gen/matplotlib.pyplot.close.html). Since we create 10 figures in this test, maybe this will only close the last one? 

Perhaps a test that explicitly closes each figure (e.g., https://github.com/earthlab/earthpy/pull/244/files#diff-dca3c53fc048765fa458bf9c0d489975) might be better, because we can be sure each figure is closed.

Reply:
just curious - how are you calling an ax object after closing a plot?

Reply:
I'm not sure I understand the question - the ax object is appended to a list `axs` before closing the plot in the above link.